cbdr/cortex
# COR-780: Click out of Popup to Close it

## Purpose:
Adds the ability for Users to click the outside of the Media Popup to close it.

> I want to be able to get out of the 'Select media' screen when I create a blog, without having to scroll and manually click close. I want to click anywhere outside of the pop up to exit this menu.

## JIRA:
https://cb-content-enablement.atlassian.net/browse/COR-780

## Steps to Take On Prod
* Go to Employer Blogs
* Select featured media
* Click the blurred area around the popup to close it
* Reopen the popup
* Close it the conventional way (click the button)
* Now confirm that the popup doesn't close by clicking around inside of it (click anywhere that's not blurred or a close button and confirm it stays open)

## Changes:
* Changes to setup
  * N/A

* Architectural changes
  * N/A

* Migrations
  * N/A

* Library changes
  * N/A

* Side effects
  * N/A

## Screenshots
* Before
N/A

* After
N/A

## QA Links:
http://web.cortex-4.development.c66.me/

## How to Verify These Changes
* Specific pages to visit
  * Employer Blog Creation

* Steps to take
  * Select featured media
  * Click the blurred area around the popup to close it
  * Reopen the popup
  * Close it the conventional way (click the button)
  * Now confirm that the popup doesn't close by clicking around inside of it (click anywhere that's not blurred or a close button and confirm it stays open)

* Responsive considerations
  * N/A

## Relevant PRs/Dependencies:
N/A

## Additional Information
N/A
